doc/nrf/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_suit_fetch.rst

```diff
@@ -132,9 +128,8 @@ To reconfigure the sample to use the fetch model, complete the following steps:
            - suit-directive-fetch:
              - suit-send-record-failure
 
-     When the secure domain firmware processes the ``suit-install`` sequence, this sequence of directives instructs the secure domain to search for a payload with a given URI in all cache partitions.
-     If no such payload is found, the update process ends with an error.
-
+      When the secure domain firmware processes the ``suit-install`` sequence, this sequence of directives instructs the secure domain to search for a payload with a given URI in all cache partitions.
+      If no such payload is found, the update process ends with an error.
```

Comment:
It looks like the indentation is wrong in the rendered doc. The `#. Modify the ``suit-install`` sequence...` and `#. Remove the application binary...` should be at the same level as `#. Add the ``suit-payload-fetch`` sequence`

Reply:
Okay, I think it is addressed now. Sometime doc-build link doesn't reflect latest changes. We can approve now, and I can re-check before RC2 deadline

Reply:
Just checked, I think the change is reflected now: https://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-15651/nrf/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_suit_external_memory.html#enabling-external-flash-support-in-suit-dfu

Reply:
Ah, I think I have corrected it now in the my latest push - hope latest doc build will show it and it looks good.